### PR TITLE
MATE-2329 New Marketing icons

### PR DESCRIPTION
**Pull request by matiasfha**
MATE-2329

### CODE REVIEW

File: src/react/helpers.ts [hunk 1]
```diff
@@ -69,6 +70,7 @@ import Champagne from  './_icons/Champagne';
 import ChangingRoom from  './_icons/ChangingRoom';
 import Chat from  './_icons/Chat';
 import Check from  './_icons/Check';
+import Chrismas from  './_icons/Chrismas';
```
Comment (Tisks) [inline src/react/helpers.ts:73]: I spotted a typo here!, please replace

File: src/react/_icons/Halloween1.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Icon from '../components/Icon';
+import { IconComponent } from '../types';
+
+const Halloween1: IconComponent = ({ fill = 'currentColor', ...props }) => (
```
Comment (Tisks) [inline src/react/_icons/Halloween1.tsx:5]: Is there any better naming for this?

**Review by Tisks — CHANGES_REQUESTED** [review]
Chrismas Typo and suggestion

File: src/react/_icons/Halloween1.tsx (hunk 2 above)
Comment (matiasfha) [reply src/react/_icons/Halloween1.tsx:5]: Not sure, this is how the original icon was named

**Review by Tisks — APPROVED** [review]

**Comment by Tisks** [comment]
I would rename `Halloween1` to `HalloweenSpider` but apart of that, 👍

**Pull request merged**